website/src/components/SearchBar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+// import {useState} from 'react'
+
+function SearchBar(props) {
```

Comment:
Please change `props` to `{ onChange }` and update the use of `props` throughout the component accordingly.